go/tools/builders/env.go

```diff
@@ -49,6 +62,31 @@ func abs(path string) string {
 	}
 }
 
+func absoluteCgoFlags(flags []string) []string {
```

Comment:
This is needed in general because it seems a lot of CROSSTOOLs are built using relative paths, which go won't understand since it cd's in $WORK and runs commands from there...

About the flag list, indeed it's difficult to have a complete list, so I figure we'd just add them as needed later on. So far I haven't needed more (and I'm building for iOS and Android).

Reply:
Fixed in the new pushed version